src/apps/LandingRegister/LandingRegister.react.js

```diff
@@ -89,6 +95,7 @@ export default Radium(React.createClass({
               style={styles.emailTextfield}
               value={this.state.textFieldValue}
               type='email'
+              onChange={this.handleTextFieldChange}
```

Comment:
You can use here "valueLink" instead of writing it by yourself (https://facebook.github.io/react/docs/two-way-binding-helpers.html). It starting to troublesome when you need something more than just updating state, but I guess in that case it can work.